packages/backend/src/open_payments/quote/service.test.ts

```diff
@@ -466,6 +470,32 @@ describe('QuoteService', (): void => {
       ).resolves.toEqual(QuoteError.InvalidReceiver)
     })
 
+    test('fails on non-positive receive amount from quote', async (): Promise<void> => {
+      const receiver = await createReceiver(deps, receivingWalletAddress)
+
+      jest
+        .spyOn(paymentMethodHandlerService, 'getQuote')
+        .mockImplementationOnce(() => {
+          throw new PaymentMethodHandlerError('Failed getting quote', {
+            code: PaymentMethodHandlerErrorCode.QuoteNonPositiveReceiveAmount,
+            description: 'Non positive receive amount for quote'
+          })
+        })
+
+      await expect(
+        quoteService.create({
+          walletAddressId: sendingWalletAddress.id,
+          receiver: receiver.incomingPayment!.id,
+          method: 'ilp',
+          debitAmount: {
+            value: 2n,
```

Comment:
Why does that end up being non-positive? Because of slippage?

Reply:
We mock the call to `paymentMethodHandlerService.getQuote` so this can be anything